### ISSUE TITLE
GPU power issue RTX 3090

### ISSUE THREAD

**Issue by Shah7877** [trexminer/T-Rex#1118]
Lately my RTX 3090 has been having power issues. Temps are normal and sitting pretty low for a 3090. But for some reason it won't use the max power output it is rated for. MSI Overclock also has the power limit set to 100% but the 3090 just decides to ignore it. Any help?

As of writing this, RTX 3090 is at 62C/82C P: 224W F:100%. I have tried it with another power supply rated at 750W just like my original power supply but nothing changed.

**Comment by UNAecho** [comment]
I 've the same issue.My RTX 3090 and MSI Overclock also has the power limit set to 100% or even bigger, but the 3090 seems to use a lower power to mine.
My 3090 has only 55MH/s.and power is 180W.

**Comment by UNAecho** [comment]
> Lately my RTX 3090 has been having power issues. Temps are normal and sitting pretty low for a 3090. But for some reason it won't use the max power output it is rated for. MSI Overclock also has the power limit set to 100% but the 3090 just decides to ignore it. Any help?
> 
> As of writing this, RTX 3090 is at 74 MH/s 62C/82C P: 224W F:100%. I have tried it with another power supply rated at 750W just like my original power supply but nothing changed.

Have you update nvida drive before your issue?
I notice i have update the nvida drive 511.79(02/14/2022) and my hash rate went down.

**Comment by Shah7877** [comment]
> 

my problem started a few months ago. I am not sure if there was an Nvidia update that caused this problem. Haven't messed around with the drivers to see if that was what caused the problem.

**Comment by Shah7877** [comment]
> > Lately my RTX 3090 has been having power issues. Temps are normal and sitting pretty low for a 3090. But for some reason it won't use the max power output it is rated for. MSI Overclock also has the power limit set to 100% but the 3090 just decides to ignore it. Any help?
> > As of writing this, RTX 3090 is at 74 MH/s 62C/82C P: 224W F:100%. I have tried it with another power supply rated at 750W just like my original power supply but nothing changed.
> 
> Have you update nvida drive before your issue? I notice i have update the nvida drive 511.79(02/14/2022) and my hash rate went down.

just tried an update from March 30, 2021 the driver version was 465.89. Didn't seem to fix the issue.

**Comment by UNAecho** [comment]
> > > Lately my RTX 3090 has been having power issues. Temps are normal and sitting pretty low for a 3090. But for some reason it won't use the max power output it is rated for. MSI Overclock also has the power limit set to 100% but the 3090 just decides to ignore it. Any help?
> > > As of writing this, RTX 3090 is at 74 MH/s 62C/82C P: 224W F:100%. I have tried it with another power supply rated at 750W just like my original power supply but nothing changed.
> > 
> > 
> > Have you update nvida drive before your issue? I notice i have update the nvida drive 511.79(02/14/2022) and my hash rate went down.
> 
> just tried an update from March 30, 2021 the driver version was 465.89. Didn't seem to fix the issue.

I used a 3070 laptop to mine and the hash rate is normal.
But my 3090 in pc is 50MH/s,I can't find the answer to the question.

I noticed that the memory was normal(10000MHz) at t-rex startup, but only a few seconds,then is went down to 5000Mhz ,so I guess it's some artificial rule that leads to the lack of hash rate

**Comment by 420Communications** [comment]
What are you guys mining? i get a stable 120Mh running 60/106 - 85% fan and 83% fan power i know its hot but from reading to get good mh for eth its normal to be above 100

**Comment by Shah7877** [comment]
> What are you guys mining? i get a stable 120Mh running 60/106 - 85% fan and 83% fan power i know its hot but from reading to get good mh for eth its normal to be above 100 and i should add, that even changing the memory by 50 etc will affect the entire OC. so it just sounds like your OC setup isn't correct if your mining ETH - windows 11 and newest always updated nvidia drivers with out issues..

I’m mining Etherum on the ETHash algorithm using the latest version of T-Rex Miner. I used to have the same temps a little lower with memory temps but had the same MH/s but all of a sudden one day it decided to force itself in a lower power usage mode.  Heckled everything with the 3d settings in the nvidia control panel nothing has worked to fix it

**Comment by 420Communications** [comment]
ok, your not really giving much info though. you should not have touched the 3d settings. that should be default.
What is OC at? in afterburner? im running -400 +1100
If that is all correct open Task Manager - check what else is using your GPU when your using the miner - that will be the issue -

I have 1 card dedicated mining rig - 1 card in my gaming rig - I 24/7 mine the gaming rig - but chrome Streaming tv puts it down to about 110 - when i game it drops to 40-50. that is a 1080p game, Mining and TV streaming on chrome. 

It could be your PC detecting the app as like xbox live or something but task manager should tell you

**Comment by Shah7877** [comment]
> ok, your not really giving much info though. you should not have touched the 3d settings. that should be default. What is OC at? in afterburner? im running -400 +1100 If that is all correct open Task Manager - check what else is using your GPU when your using the miner - that will be the issue -
> 
> I have 1 card dedicated mining rig - 1 card in my gaming rig - I 24/7 mine the gaming rig - but chrome Streaming tv puts it down to about 110 - when i game it drops to 40-50. that is a 1080p game, Mining and TV streaming on chrome.
> 
> It could be your PC detecting the app as like xbox live or something but task manager should tell you

OC settings are all set to default other than fan speed set to 100% and  that’s just to force the card to use more power. No other apps are running on the computer and when I game I don’t run the miner at the same time because I don’t want to risk damaging the card even more. I’m gonna try a fresh install of windows soon and see if that fixes the issue. I have 2 other rigs that have never been touched since the release of the 3090 and they are working perfectly fine. This is my daily rig that I use to game and mine with so maybe something I downloaded is using the gpu or something I have no idea but I think a fresh install of windows might fix the issue.

**Comment by 420Communications** [comment]
:s sorry your trying to OC with out changing the OC settings in afterburner?

**Comment by 420Communications** [comment]
![image](https://user-images.githubusercontent.com/69356161/156446623-38e7c1bb-8968-4d73-a6d5-7943ba6075ac.png)

**Comment by Shah7877** [comment]
> :s sorry your trying to OC with out changing the OC settings in afterburner?

Yep everything is set too default. Even if I apply my OC settings which was -125 core and +1000 memory it has no effect on the card and actually makes it preform even worse

**Comment by 420Communications** [comment]
correct because you need to set it right. I said above in one comment you can chance the core clock by 50 and it will stuff the entire oc up make it run like dog balls - set to my settings above. try it.

**Comment by Shah7877** [comment]
> correct because you need to set it right. I said above in one comment you can chance the core clock by 50 and it will stuff the entire oc up make it run like dog balls - set to my settings above. try it.

I think the issue lies more than just OC settings. Don’t get me wrong the settings you are running at are perfect but the card out of the box should be running max TDP if it is set to 100% power usage abs it should at least get 95 MH/s without even touching the settings. So in my opinion I don’t think changing my OC setting to yours would change anything with the power usage.

**Comment by 420Communications** [comment]
![image](https://user-images.githubusercontent.com/69356161/156448228-d5f6b18d-bd60-40db-8d84-0d8668997137.png)

This is mine running right now mining and streaming discovery+ power wont run at 100% but utilization should be.

**Comment by Shah7877** [comment]
> ![image](https://user-images.githubusercontent.com/69356161/156448228-d5f6b18d-bd60-40db-8d84-0d8668997137.png)
> 
> This is mine running right now mining and streaming discovery+ power wont run at 100% but utilization should be.

Yep just checked my HWMonitor. Nothing else on the PC is running except the miner and it’s reporting 100% utilization but power is at 231.68W with a max of 237 and min of 221

**Comment by 420Communications** [comment]
Is that with my OC settings? if not - put to my settings run the miner - then check HW monitor - it should or will be lower if your on teh PC using it whilst its mining as well - literally 1 web browser open will slow it down. 
A windows reinstall will work - its just overkill - this shouldnt be that hard to fix - I have run into these problems but its been mostly to do with Afterburner and me changing settings.

**Comment by Shah7877** [comment]
> Is that with my OC settings? if not - put to my settings run the miner - then check HW monitor - it should or will be lower if your on teh PC using it whilst its mining as well - literally 1 web browser open will slow it down.
> A windows reinstall will work - its just overkill - this shouldnt be that hard to fix - I have run into these problems but its been mostly to do with Afterburner and me changing settings.

I’ll give it a shot with yours. Yeah a reinstall is overkill for sure but it def fixes the problem. But I’ll respond later today after trying the OC settings and give you an update. Preciate the help!

**Comment by 420Communications** [comment]
How did you go?

**Comment by Shah7877** [comment]
> How did you go?

Trying it out right now sorry for the late reply. Currently the hashrate is actually going down as im adjusting the core clock and memory clock.

**Comment by Shah7877** [comment]
> How did you go?

-400 on the core clock and +1000 on the memory. Nothing has changed. If anything my power usage went down to 216W and im actually getting a worse hashrate. Its currently at 63 MH/s

**Comment by 420Communications** [comment]
ok, as you do not wish to follow what I'm asking you to do, I will let someone else try assist you.

I have already said you change something by even 50 (memory) and you will lose hashrate.
For some reason you are refusing to follow the settings for OC that is on about 50+ forums confirmed working. 

I change mine to 1000 and it makes my hashrate drop down to 48. So your getting better then me using incorrect settings. It has to be set to my settings (fan speed and power can be really anything you want) - but the actual main settings for the 3090 for stable 120 100% of the time it has to be what i linked

**Comment by Shah7877** [comment]
> ok, as you do not wish to follow what I'm asking you to do, I will let someone else try assist you.
> 
> I have already said you change something by even 50 (memory) and you will lose hashrate. For some reason you are refusing to follow the settings for OC that is on about 50+ forums confirmed working.
> 
> I change mine to 1000 and it makes my hashrate drop down to 48. So your getting better then me using incorrect settings. It has to be set to my settings (fan speed and power can be really anything you want) - but the actual main settings for the 3090 for stable 120 100% of the time it has to be what i linked

I literally used the same settings that you had in the picture you posted on this thread. I dont know why you are getting so heated other than just trying to help someone who is obviously new to the mining community/overclocking community. Not everyone you help on here is going to automatically know what exactly to do so go ahead bud I really dont care if you give up lmao

**Comment by 420Communications** [comment]
I understand that mate. But maybe mention that then. 
Most people that get into mining ie: spend thousands on items to get into it - they should have some basic idea, researched etc..

The way your explaining yourself, you want your card to run at full power 100% of the time when noone wants that. Mining is about getting the most MH for the least amount of power. That is litrally the fundamentals of mining fun or to make money - because even for fun your learning to get the most for less. 

You have another OC program installed - one that came with the Card - depending on the card it could be like me the gigabyte v tuner or something else - it sounds like its taking over - if you have set to my settings - waited atleast 15 minutes and see that it doesnt reach 120. Now this is with litrally not a single thing running on the PC - if you open 1 Chrome tab 120 MH will drop to 110 MH. open a second tab and your going to be at 100 MH.
Steam something HD you can be down to 50 MH
are you touching the PC? have you got items in task manager running? etc.. 

Did not mean to have a go at you, Just yeah if your doing technical stuff, like mining its wroth saying hey im not great with this etc. so we dont treat you like you know what your doing and can better try assist you .

**Comment by 420Communications** [comment]
there is also the question of, where are you seeing the Hashrate? 

The T Rex Console? The T Rex Web UI? or your exchange?

**Comment by Shah7877** [comment]
I'm not trying to have it run at the standard full power it is rated for. I used to run at 115 MH/s with around 275W which was the first time I ever started out with overclocking and mining. This was with a -250 on core clock and +900 on memory clock. and a power limit of 80%. Now the card is stuck on 0 for core and memory clock with 100% power limit and doesnt draw full power at all even when i do a 3DMark stress test on the card. 

But yes I kept your settings running for a little while and nothing changed. No increase in Mh at all and no change in the power draw from the card. Might do the clean restart of the OS or maybe its something to do with the apt complex I live in and the amount of power draw its getting from the wall. 

This problem didnt start until moving here compared to living at home with my parents where everything was fine there. Its not even connected to a power extender or anything. I had nothing running on my computer while tweaking with the overclock other than the mining program, MSI Afterburner which is the only overclocking tool I have ever used for my gpu, and HWMonitor to monitor anything else. The 3090 is a PNY Uprising EPIC-X with the thermal pads being replaced due to the manufacturer having faulty thermal pads that didnt allow the card to be overclocked at all whenever I first got it. Temps were terrible until i replaced thermal pads and thermal paste for the card.

**Comment by Shah7877** [comment]
> there is also the question of, where are you seeing the Hashrate?
> 
> The T Rex Console? The T Rex Web UI? or your exchange?

I am looking at the T-Rex Console for the megahash. The .bat file i open to start the miner to be exact. Etherpool is the pool i mine from and it displays the same MH/s in the reported hasharate section but obviously the current hashrate area changes depending on the valid shares.

**Comment by 420Communications** [comment]
and 100% you dont have velocity x  installed? 

If not should try it. I know PNY isnt a bad brand - its just not something i touched before - so could be touchy.
